book/plugins/general-abilities/js2native/readme.md

```diff
@@ -122,26 +137,29 @@ Action which is set to “Area Switched".
  *best practice* - use the product name as the "Category". (i.e. "Feed", or "Stitcher")
  **properties** - The corresponding properties of the event key. The keys of the
 JSON object should be formatted as pascal case (*PascalCase*).
+ 
  ```javascript
```

Comment:
@snooky23 something got wrong in the markdown here.
![Screen Shot 2019-07-15 at 10 59 20](https://user-images.githubusercontent.com/3192941/61202427-25ea8880-a6f0-11e9-8363-6c8c920e1362.png)

Reply:
fixed